src/components/POLine/LineLinkedInstances/hooks/useLinkedInstances/useLinkedInstances.js

```diff
@@ -1,4 +1,5 @@
 import { useQuery } from 'react-query';
+import { keyBy, uniq } from 'lodash/fp';
```

Comment:
Could you please split to several lines example below?
```
import { 
  keyBy,
  uniq,
} from 'lodash/fp';
```

Reply:
done